app/views/external_users/claims/confirmation.html.haml

```diff
@@ -17,5 +17,6 @@
         = render partial: 'shared/disk_evidence_info', locals: { f: claim }
 
       .button-holder
+      .govuk-button-group
```

Comment:
Should this be nested inside `button-holder`?

Reply:
Thank you Joe. You are right. I have nested it inside button-holder and it seems to format correctly.